### ISSUE TITLE
[Question]: Is there a way to get confidence scores in the answers in ERNIE-LAYOUT for MRC format? 

### ISSUE THREAD

**Issue by aashishpokharel** [PaddlePaddle/PaddleNLP#6656]
I want to get the **confidence score** for the answers in ERNIE-LAYOUT (similar to the TaskFlow demo & LayoutLM pipeline). But I cannot find the way to do it in "mrc" format.
Is there a way to implement that? 
Context: I want to surf through a number of pages in a pdf and  find the answer for a question with best confidence score among all the pages.

**Comment by aashishpokharel** [comment]
Found the plausible solution to this, You just need to Normalize the start_logits and end_logits and pass it through the softmax to get confidence scores for the starting index and end_index. The simply doing avg(start_logits, end_logits) or start_logits * end_logits will do the job.
